data-imports/import_hpd_violations.py

```diff
@@ -0,0 +1,198 @@
+import calendar
+import clean_utils
+import datetime
+import luigi
+import luigi_utils
+import os
+import tempfile
+import utils
+
+class DownloadHPDViolations():
```

Comment:
Small nitpick, but we [should be using new-style classes](https://stackoverflow.com/questions/54867/what-is-the-difference-between-old-style-and-new-style-classes-in-python) the inherit from `object`